core/blockchain_test.go

```diff
@@ -2345,7 +2345,7 @@ func testInsertKnownChainData(t *testing.T, typ string, scheme string) {
 			}
 		}
 	} else {
-		inserter = func(blocks []*types.Block, receipts []types.Receipts) error {
+		inserter = func(blocks []*types.Block, _ []types.Receipts) error {
```

Comment:
I feel this change is a bit unnecessary. We have a tons of unused parameters, e.g. for implementing some interfaces.

I am worried people will submit endless prs to change these things.

Reply:
yes, it is a bit unnecessary. And test ci has already checked this functions. I have restored this functions.